server_http/templates/index.html

```diff
@@ -12,8 +12,34 @@
 <body>
     <div class="gridContainer">
 
-        <input class="nameInputGrid" id="login" placeholder="Your Name">
-        <input class="chatInputGrid" id="findChat" placeholder="Chat Name" value="{{ requestedChatName }}">
+        <div class="nameInputGrid">
+            <input id="login" placeholder="Your Name">
+            <i class="tooltip" id="loginNameInfo">&#9432;
+                <span class="tooltipText">Username restrictions:
+                    - at least 1 character long
+                    - at most 20 characters long
+                    - can contain basic Latin characters (lowercase / uppercase) and numbers
```

Comment:
Verification functions allow also special diacritic characters, don't them? Isn't this tooltip a bit misleading?

Reply:
Updated the tooltip's description